product.html

```diff
@@ -141,5 +144,200 @@
+                      fill="#363535" />
+                  </g>
+                  <defs>
+                    <clipPath id="clip0_2923_446">
+                      <rect width="16" height="16" fill="white" />
+                    </clipPath>
+                  </defs>
+                </svg> Retiro en Tienda
+              </button>
+              </div>
+            </form>
+          </div>
+          <form action="" method="post" class="product__add-card">
+            <div class="product__add-card-counter">
+              <button onclick="restar()" type="button" class="product__add-card-counter-button product__add-card-counter-button--less-add">
+                <svg xmlns="http://www.w3.org/2000/svg" width="26" height="30" viewBox="0 0 26 30" fill="none">
+                  <path d="M5.9574 15H20.4539" stroke="#D8D8D8" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                </svg>
+              </button>
+              <input class="product__add-card-number js-product__add-card-number" type="number" name="quantity" value="1"></input>
+              <button onclick="sumar()" type="button" class="product__add-card-counter-button product__add-card-counter-button--more-add">
```

Comment:
Preferiblemente no uses onclick u otro evento de javascript en el html, es mejor asignarlo en el archivo .js para tener las cosas mas ordenadas
```suggestion
              <button type="button" class="product__add-card-counter-button product__add-card-counter-button--less-add js-add-unit-to-product">
```